addons/l10n_do/models/chart_template.py

```diff
@@ -0,0 +1,23 @@
+# coding: utf-8
+# Copyright 2016 iterativo (https://www.iterativo.do) <info@iterativo.do>
+
+from openerp import models, api, _
+
+
+class WizardMultiChartsAccounts(models.TransientModel):
+    _inherit = 'wizard.multi.charts.accounts'
+
+    @api.model
+    def _get_default_bank_account_ids(self):
+        return [
```

Comment:
It actually was `if self.env.user.company_id.country.code_id.upper()` "_id" was missing.